config/locales/cy/privacy_policy.yml

```diff
@@ -2,146 +2,101 @@
-        list: |2-
-
-          ot su seriuqer wal eht
-          tnemucod siht ni tuo tes sesoprup eht rof dedeen si ti
-        info_html: .reciffO noitcetorP ataD ruo morf ypoc a tseuqer ro ,>a/<atad gniniater
-          no ycilop ruo>"fdp.9102-tsugua-sdrr-aal/035428/elif/atad_tnemhcatta/sdaolpu/metsys/sdaolpu/tnemnrevog/ku.vog.ecivres.gnihsilbup.stessa//:sptth"=ferh
-          a< weiV
-      rights:
-        header: sthgir ruoY
-        list_heading: ":tseuqer ot thgir eht evah uoY"
-        list: |2-
-
-          atad lanosrep ruoy eteled ro esu ,tcelloc ot ffats tcurtsni ew woh no noitamrofni
-          snoitasinagro rehto htiw evah ew stnemeerga gnirahs-atad yna fo ypoc a
-          ti rof noitacifitsuj a regnol on si ereht fi desare si atad lanosrep ruoy taht
-          detcerroc si atad lanosrep ruoy ni etaruccani gnihtyna taht
-          )'tseuqer ssecca tcejbus' a( atad lanosrep taht fo ypoc a
-          dessecorp si atad lanosrep ruoy woh tuoba noitamrofni
-        info_html: ".reciffO noitcetorP ataD >rbba/<)JOM(>'ecitsuJ fo yrtsiniM'=eltit
-          rbba< ecitsuJ fo yrtsiniM eht tcatnoc ,stseuqer eseht fo yna evah uoy fI"
-        data_protection_office_contact_details:
-          title: reciffO noitcetorP ataD ehT
-          email_address_html: '>a/<ku.vog.ecitsuj@ycavirp>"ku.vog.ecitsuj@ycavirp:otliam"=ferh
-            a<'
-          address:
-            line_1: ecitsuJ fo yrtsiniM
-            line_2: ecnarF ytteP 201
-            line_3: nodnoL
-            post_code: JA9 H1WS
-      contact_or_complaint:
-        header: tnialpmoc a ekam ro su tcatnoC
-        data_protection_office:
-          list_heading_html: ":uoy fi reciffO noitcetorP ataD >rbba/<JOM>'ecitsuJ
-            fo yrtsiniM'=eltit rbba< eht tcatnoC"
-          list: |2-
-
-            ro desusim neeb sah atad lanosrep ruoy taht kniht
-            eciton ycavirp siht ni gnihtyna tuoba noitseuq a evah
-          contact_details:
-            email_address_html: '>a/<ku.gro.oci@krowesac>"ku.gro.oci@krowesac:otliam"=ferh
-              a<'
-            telephone: 3111 321 3030 :enohpeleT
-            text_phone: '068545 52610 :enohptxeT'
-            call_hours: mp03:4 ot ma9 ,yadirF ot yadnoM
-            call_costs_html: '>a/<segrahc llac tuoba tuo dniF>"segrahc-llac/ku.vog.www//:sptth"=ferh
-              a<'
-        commissioner_office:
-          list_heading_html: ":)>rbba/<OCI>'eciffO s'renoissimmoC noitamrofnI'=eltit
-            rbba<( eciffO s'renoissimmoC noitamrofnI eht tcatnoC"
-          list: |2-
-
-            noitcetorp atad tuoba ecivda tnednepedni rof
-            tnialpmoc a ekam ot
-          address:
-            title: eciffO s'renoissimmoC noitamrofnI
-            line_1: esuoH effilcyW
-            line_2: enaL retaW
-            line_3: wolsmliW
-            line_4: erihsehC
-            post_code: FA5 9KS
+      h1-heading: Privacy policy
+      purpose:
+        header: esopruP
```

Comment:
Is it necessary to have to manually reverse all the privacy policy information here or can I just get away with this for now as it will be updated in actual Welsh later?